[WIP] 1005/chapter/remote events (#1008)
alan-turing-institute/the-turing-way: book/content/remote_events/01/activity-types.md

# Activity Types

- [Panel discussion](#panel-discussion)
- [Presentation/Talk Sessions](#presentationtalk-sessions)
- [Prototyping or Ideas Generation Session](#prototyping-or-ideas-generation-session)

## Presentation/Talk Sessions

A presentation or talk is an activity where one person addresses a room on a topic using a slide deck as a means of supporting or expressing their point.

> Talks can be given in a variety of styles, for example:

Author: ```suggestion
Talks can be given different styles, for example:
```

# Activity Types

- [Asyncronous Collaboration on a Text Document](#asyncronous-collaboration-on-a-text-document)
- [Panel discussion](#panel-discussion)
- [Poster Sessions](#poster-sessions)
- [Presentation/Talk Sessions](#presentationtalk-sessions)
- [Prototyping or Ideas Generation Session](#prototyping-or-ideas-generation-session)
- [Workshops and Tutorials](#workshops-and-tutorials)

> ## Asyncronous Collaboration on a Text Document

Reviewer: Spl - asynchronous

By asynchronous collaboration on a text document we mean where 2 or more people write a document together such as a paper, technical report, funding application or blog post not simultaneously.

### Requirements

Usually, for asyncronious collaborative document writing the authors will continiuosly add to a document going through phases of brainstorming, planning, writing, spell/grammar checking, proof reading and editing.
Not all of these phases will actually link to the document but may include meetings, phone calls, emails, mind mapping and diagram/image creating tools.
The authors will need to keep track of the version that they are working on, who has contributed what and what has been proof read and edited by who and when.

> Some documents such as papers and funding application may also require strick formatting and word/character counts.

Reviewer: Spl - strict formatting

But for online asynchronous collaboration on a text document the main focus is on writing and people tend to forget the other related activities.
For generating text the most important functionality is the formatting and templating whether this fits with the final destination of the document.
Mathematicians use systems that allow for the good formatting of mathematical notations such as latex while those who do not use matematical notations may prefer a traditional word processor.

> Along with this it is important to have accessibility features such as alter the font, background colour, read aloud as well as spell and grammar checking.

Reviewer: spelling rather than spell?

Now we can add features outside of the actual text document such as talking to the other authors, planning and structuring the document (eg mind mapping) and creating graphs/diagrams/figures.

### Solutions

There are several different

#### Available tools

## Panel discussion

A panel discussion is an activity where a group of people -- the *panel* -- discuss a topic in front of an audience.
Panel discussions usually address a specific topic, and the panel will often include people with knowledge on the topic that is discussed.

At in-person events, panel discussions are usually led by a moderator, who has a number of tasks:

- Introducing the topic that is being discussed;
- Introducing the members of the panel;
- Explaining the rules of the discussion to the panel and the audience,
such as the order in which members are asked to answer a question, or time limits and other rules for giving answers;
- Moderating the discussion, for example by asking specific people specific questions.

After some rounds of questions from the moderator to the panel, the discussion is often opened up to the audience, members of which can then ask the panel further questions.

> It is, agagin, the task of the moderator to pick the people to ask questions, and to moderate answers.

Reviewer: Spl - again

[sli.do](https://www.sli.do/) is a platform for to collect questions for Q&A sessions.
Users login to their event that can be split into sessions relating to each talk.
They can then submit questions which can be upvoted by other users.
A session chair can then moderate these question and pose them to the speaker.
[Mentimeter](https://www.mentimeter.com/) is a more full-fledged presentation platform which also includes Q&A functionality where the audience can ask questions, and vote on questions.

## Poster Sessions

By a poster session we mean an event where posters on a similar topic are presented together at a larger event such as a conference.
To some extent they are a networking activity as one or more authors of the poster stands by the poster during the session to talk to conference participants.

### Requirements

> Usually poster sessions are a good way for early career researchers to present their work and become comfortable with their community although researchers of any career stage should publish posters.

Reviewer: community, although - I think!

Often a conference will give a call for posters and an abstract for the content of the poster will be provided for review.
The conference will provide a format which is often one page of A1 paper.
At some conferences the poster or the abstract will count as a publication and be included in the proceedings.
It is common for the conference organisers to place the poster session in an area where there is a high footfall so that the posters are seen by a large proportion of those who attend the conference.
It is becoming popular to give the authors one minute to present the information in their poster to the conference.

> But for online events the social elements of psoter session are much more difficult to recreate.

Reviewer: spl - a poster

Other benefits include support for a large number of participants, breakout rooms to facilitate discussions between smaller groups, and a chat feature.
- [sli.do](https://www.sli.do/) is a platform for to collect questions for Q&A sessions.
Users login to their event that can be split into sessions relating to each talk.
They can then submit questions which can be upvoted by other users.
A session chair can then moderate these question and pose them to the speaker.

## Prototyping or Ideas Generation Session

By a prototyping or an ideas generation session, which in UK academia can also be termed a sandbox, we mean a group activity focused on brain storming around a theme which can a particular problem or set of problems or on how to get the most of a new technology, technique or material etc.
These activities tend to also act as a way to introduce people who have shared concerns but will not previously have met as they are from different disciplines or communities.

### Requirements

Usually, at a prototyping or an ideas generation events you have a lead or chair who co-ordinates the activity.

> The session will start with each person introducing themselves and providing details of where their interests are relevant to the theme of the session. At small events where the participants may also be asked to bring an item or items that represents them or the topic that interests them.

Reviewer: Two sentences on a line

At a larger event there may be too many participants to allow everyone to interduce themselves in which case there will be an introduction and selected participants will give a presentation.

Once everyone understands the theme and has been introduced there will a networking session or breakout sessions.
Generally, at this point the chair will identify topics or sub-themes and note them down so that all can see.

A networking session allows people to mingle and find other participants with shared interests.
If there are good common themes or sub-themes then the chair may help participants find participants with shared interests.
This could be down by putting a flip chart of post-its in different parts of a room so that participants can cluster and talk.

Alternatively break out sessions can be more structured and allow the participants to talk, do an activity such as making notes for a blog or notes for post-its or drawing pictures of products or art works.
More creative sessions may provide a maker space or materials to build prototypes.

> The break-out groups can be self-organised from the ‘grass roots’ or the chair may allocate participants to groups and may event move them between a number of break out groups and sessions so that for example every participant is shares at least 1 break out session with every other participant.

Reviewer: spl - even move them...
so that, for example, every participant shares at least...

At the end of the event there will be a wrap up session where all the participants get together to share the results of the networking or breakout session.
Each break out group or participant is asked to share the ideas that they generated to the whole group.
Once complete the top ideas are selected by the group or by the chair.
This often ends with a networking session where participants can share their contact details so that they can continue working together on ideas that have come out of the event.

The session can end here however if the participants want further involvement then in follow up activities are useful.
These are especially important if the original event was aimed at generating ideas for a funding call or art exhibition.
Follow on activities may include making the generated ideas and the participants contact details online available to all who want to continue; or a coffee morning; or a weekly meet-up.

But for an online prototyping or ideas generation event you need a variety of online meeting areas and tools.

> Networking is a social activity but in this case the participants will need help other useful participants.

Reviewer: I think there is a missing word here

Break-outs will need other online collaborative working tools such as for note taking, programming, graphics such as CAD files, 3D printing files or photographs.
Co-ordinating the booking and sharing all of the necessary online tools along with information about them with each participant and break-out group is challenging especially as often digital resources do not have meaningful identifiers and may be supplied by a different vendor with different licensing rules.
These types of events work best with small groups as participants get to know each other better and the size of the group is an important consideration when you go online.

### Solutions

There are several different types of online tools required for this type of event.

- Online meetings systems with the capacity for many break-out or sub-sessions

> - Collaborative working tools - generally speaking these will be for text, white borads or story boards however specialist communities may want to use all sorts of tools depending on the time they are alloted.

Reviewer: white boards

  A long time is similar to a hack event where things other than coding may be done.
- Sharing concept and ideas eg picture boards (piminterest)
- Navigation tools to allow the chair and participants to co-ordinate their activities easily

#### Available tools

## Workshops and Tutorials

> By a workshop of tutorial we mean a training event on a particular topic that runs typically from 90mins up to several days.

Reviewer: workshop or tutorial

> Normally these will require more than a teacher lecturing but involve hands on work by the participants and so require that the participants understand what level of skills they have to know that they fit the prerequisits for the event.

Reviewer: spl - prerequisites

If the student is using their own computer then they may need to know information on system requirements and how it should be configured.

### Requirements

Usually at a workshop or tutorial there will be a combination of presentations and exercise sheets for the students to work from.

> Work shop or tutorials provided as a training event at work probably will be delivered using in a work environment will probably use a cluster where a system adminstrator has configured and installed the software and then the teacher tests the set up and makes sure the right versions of the materials and data are available to the students.

Reviewer: Workshop or tutorial...
Also do you mean a work computing environment?

> This tends to reduce any problems during the delivery of the event but means that students struggle after as they amy not be able to re-create the required environment.

Reviewer: spl - they may not be able...

> For large groups there may be assistants to help the lead and the students students if they get stuck.

Reviewer: spl - help the leader and the students if they...

> If the workshop uses software with a complicated user interface or grapical views then the teacher will need to project thier screen so that all the students can see it.

Reviewer: spl - graphical views...project their screen

But for online events the workshop or tutorial can move into the cloud where the student uses a virtual environment that is configured by the teacher before hand.
In the best case the student will be able to continue using this virtualised environment online after the event or download the configuration for this environment and install it in a virtual manager system such as VirtualBox.
The data and other materials can be provided online and/or the students can download so they have a local copy.

> Presentations can be given through online meeting systems while complex unser interface actions can be captured through screen shot videos for the students to reference.

Reviewer: spl - user interface

When a student needs help both the teacher and student may need to share their screen with each other.

### Solutions

There are several different types of tools and environments needed for this type of event.

> Many of them are relevant to other activity types such as meetings and data stores e.g., zenodo to store data in a fully credited manor.

Reviewer: Latin - data stores, for example Zenodo to...
spl - fully credited manner
